docs/annotations.md

```diff
@@ -11,32 +11,36 @@ Example:
         openfeature.dev/enabled: "true"
 ```
 
-### `openfeature.dev/featureflagconfiguration`
-This annotation specifies the names of the FeatureFlagConfigurations used to configure the injected flagd sidecar.
+### `openfeature.dev/flagsourceconfiguration`
+This annotation specifies the names of the FlagSourceConfigurations used to configure the injected flagd sidecar.
 The annotation value is a comma separated list of values following one of 2 patterns: {NAME} or {NAMESPACE}/{NAME}. 
 If no namespace is provided it is assumed that the CR is within the same namespace as the deployed pod.
+If multiple CRs are provided, they are merged with the latest taking precedence, for example, in the scenario below, `config-B` will take priority in the merge, replacing duplicated values that are set in `config-A`.
+
 Example:
-```yaml
-    metadata:
+```
+  metadata:
     annotations:
         openfeature.dev/enabled: "true"
-        openfeature.dev/featureflagconfiguration: "demo, test/demo-2"
+        openfeature.dev/flagsourceconfiguration:"config-A, config-B"`
```

Comment:
```suggestion
        openfeature.dev/flagsourceconfiguration:"config-A,config-B"`
```
Does spacing work? In the e2e tests we don't use spacing

Reply:
https://github.com/open-feature/open-feature-operator/blob/5f839209008ced69198cdf909659c0001e1b3910/webhooks/pod_webhook.go#L223

Spacing is handled in the parse func